### ISSUE TITLE
Support mathjax on the site

### ISSUE THREAD

**Issue by deemp** [objectionary/normalizer#220]
Check mdBook [docs](https://rust-lang.github.io/mdBook/format/mathjax.html).
